fluent-react/src/localization.ts

```diff
@@ -29,13 +35,16 @@ const reMarkup = /<|&#?\w+;/;
 export class ReactLocalization {
   public bundles: Iterable<FluentBundle>;
   public parseMarkup: MarkupParser | null;
+  private reportError: (error: Error) => void;
```

Comment:
Making this private would be a breaking change, given that it's currently public.

```suggestion
  public reportError: (error: Error) => void;
```